tensorflow-ops/tests/OpsTest.hs

```diff
@@ -33,25 +33,31 @@ import qualified TensorFlow.Nodes as TF
 import qualified TensorFlow.Ops as TF
 import qualified TensorFlow.Session as TF
 import qualified TensorFlow.Tensor as TF
+import qualified TensorFlow.Types as TF
 
 -- | Test that one can easily determine number of elements in the tensor.
+testSize :: Test
 testSize = testCase "testSize" $ do
-    x <- eval $ TF.size (TF.constant [2, 3] [0..5 :: Float])
+    x <- eval $ TF.size (TF.constant (TF.Shape [2, 3]) [0..5 :: Float])
```

Comment:
I'm curious of the reason to remove the use of OverloadedLists here; maybe we can improve something in our libraries to prevent warnings?